src/main/java/mat/server/clause/FhirMeasureRemoteServiceImpl.java

```diff
@@ -37,12 +37,9 @@ public FhirConvertResultResponse convert(Result sourceMeasure) throws MatExcepti
         }
         try {
             return fhirMeasureService.convert(sourceMeasure, vsacTicketInformation.getTicket(), LoggedInUserUtil.getLoggedInUser());
-        } catch (MatException e) {
-            logger.error(e);
+        } catch (RuntimeException|MatException e) {
```

Comment:
Hm.. interesting..
Curious why the exception wrapping causes the issue..

Reply:
its logging the stack trace that is the issue here. my fix does that.